### DISCUSSION TITLE
该不该润？

### DISCUSSION THREAD

**Discussion by Diogsmhkom** [The-Run-Philosophy-Organization/run#482] [General]
普通一本金融学，有一点计算机和英语的能力，在备考本校cs学硕。
本人性格内向、自闭、孤僻，在国内朋友很少，父母是双职工，一个月加起来不到一个w。
老家是四线地级市的，虽说不发达但房价也有七八千。
家里个别人很传统，属于典型的山东孔教门徒，喜欢打压年轻人，因为性格原因在家经常被各种嘲笑打压。
现在看起来大城市房子买不起，回家躺平也躺不下，城市户口也没有地种，一想到种种问题就心力交瘁。
看到父母累死累活工作给我攒首付攒彩礼的样子我就感到很心痛。
现在就希望能去一个躺平亦有尊严的地方，等父母老了也可以把他们接过去躲躲大运动。
现在看到知乎上有人说程序员润日本，就很心动，但考虑到也是几年以后的事情了，就犹豫不定也很担心。

**Comment by wedxz** [comment]
深有同感
